tests/requirements/views/web/index.php

```diff
@@ -2,76 +2,85 @@
 /* @var $this YiiRequirementChecker */
 /* @var $summary array */
 /* @var $requirements array[] */
-?><!DOCTYPE html>
+?>
+<!DOCTYPE html>
```

Comment:
```suggestion
?><!DOCTYPE html>
```
There should be nothing before `<!DOCTYPE html>`